bin/internal/release-candidate-branch.version

```diff
@@ -0,0 +1 @@
+2.12.0-4.1.pre
```

Comment:
@CaseyHillers was this intended to point to the version, or the candidate branch?

Reply:
It should point to the branch, so then CP authors can go to something like https://github.com/flutter/flutter/blob/stable/bin/internal/engine.version to find the release branch to send CPs to

Reply:
I filed https://github.com/flutter/flutter/issues/101773 to track fixing it

Reply:
I considered writing a bot that would take the human out of the loop and handle this for us, but there seems to be some security implications.